database.types.ts

```diff
@@ -9,6 +9,60 @@ export type Json =
 export type Database = {
   public: {
     Tables: {
+      grants: {
+        Row: {
+          amount: number
+          created_at: string
+          deadline: string | null
+          description: string
+          FK_organizations: number
+          FK_orgFunded: number | null
+          id: number
+          isOpen: boolean
+          name: string
+          requirements: string | null
```

Comment:
Any particular reason to default to null? It seems like it will always be some kind of string.

Reply:
That's just how i set it when setting up the table for the first time